src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/themes/AceThemes.java

```diff
@@ -32,6 +32,9 @@
 import org.rstudio.studio.client.application.Desktop;
 import org.rstudio.studio.client.application.events.EventBus;
 import org.rstudio.studio.client.common.DelayedProgressRequestCallback;
+import org.rstudio.studio.client.server.ServerError;
```

Comment:
Super-nit-picky: update the Copyright to 2009-18. Same on other files being modified. We are all inconsistent on this, and it doesn't really matter, so revising to super-super-super-nit-picky.